### PR TITLE
Add release details to changelog

### PR DESCRIPTION
**Pull request by willingc**
@takluyver @Carreau I've grouped and added the changes in the 5.0 release.

### CODE REVIEW

**Comment by Carreau** [comment]
I kicked travis.

**Comment by willingc** [comment]
Poor travis ;-)

**Comment by Carreau** [comment]
> Poor travis ;-)

It's ok I was gentle.

File: docs/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,29 @@ Changes in Jupyter console
...
+Image Handling improvement
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+- use PIL as default image handler `#79 <https://github.com/jupyter/jupyter_console/pull/79>`_
+- better indication of whether image data was handled `#77 <https://github.com/jupyter/jupyter_console/pull/77>`_
+
+Prompts improvement
+^^^^^^^^^^^^^^^^^^^
+- use prompt_toolkit 1.0 `#74 <https://github.com/jupyter/jupyter_console/pull/74>`_
+- don't use prompt_manager `#75 <https://github.com/jupyter/jupyter_console/pull/75>`_
+- reinstate continuation prompts `#62 <https://github.com/jupyter/jupyter_console/pull/62>`_
```
Comment (Carreau) [inline docs/changelog.rst]: I merged, though I dont' think we need to be that detailed. 


`reinstate continuation prompts` seem to have been a bug that was fixed after migrating to prompt toolkit, so is not really necessary for users to know about.

Comment (willingc) [inline docs/changelog.rst]: Feel free to delete whatever details aren't relevant. I think the labels are the key thing for users not necessarily the bullets.

**Pull request merged**